samples/mpsl/timeslot/src/main.c

```diff
@@ -219,26 +238,21 @@ static void mpsl_nonpreemptible_thread(void)
 
 static void console_print_thread(void)
 {
-	uint32_t signal_type = 0;
+	uint32_t signal_action = 0;
 
 	while (1) {
-		if (k_msgq_get(&callback_msgq, &signal_type, K_FOREVER) == 0) {
-			switch (signal_type) {
-			case MPSL_TIMESLOT_SIGNAL_START:
+		if (k_msgq_get(&callback_msgq, &signal_action, K_FOREVER) == 0) {
+			switch (signal_action) {
+			case MPSL_TIMESLOT_SIGNAL_ACTION_NONE:
```

Comment:
This looks wrong. `MPSL_TIMESLOT_SIGNAL_ACTION_NONE` should not "Callback: Timeslot start"

Reply:
Yes, it is misleading. will fix it.